content_scripts/hints.js

```diff
@@ -1,5 +1,5 @@
 var Hints = (function(mode) {
-    var self = $.extend({name: "Hints", eventListeners: {}}, mode);
+    var self = $.extend({name: "Hints", eventListeners: {}, lastCreateAttrs: {}}, mode);
```

Comment:
Please use `lastCreateAttrs` as an internal variable, as we don't want this variable to be changed outside of Hints.